core/src/session.rs

```diff
@@ -390,7 +390,7 @@ impl Session {
     }
 
     pub fn set_session_id(&self, session_id: String) {
```

Comment:
Could you additionally change the type to `&str` and check where the method is used and remove possible clones?

Reply:
Dang, missed that comment just as I merged. Please submit as an additional PR - sorry and thanks!

Reply:
Yeah was a bit to slow there^^;

Reply:
https://github.com/librespot-org/librespot/pull/1422